src/components/proximities/proximities.tsx

```diff
@@ -0,0 +1,16 @@
+import { fetchProperties } from '../../api/fetch-properties';
+import { LoadingError } from '../loading/loading-error';
+import { ProximitiesSection } from './proximities-section';
+
+export const Proximities = () => {
+  const { data = [], loading, error } = fetchProperties();
+  const proximities = data?.[0]?.proximities || [];
+  const latitude = data?.[0]?.address.latitude || 0;
+  const longitude = data?.[0]?.address.longitude || 0;
```

Comment:
como vc ta passando um valor default para o data quando faz o `data = []`, vc nao precisa testar a primeira parte.

```suggestion
  const { data = [], loading, error } = fetchProperties();
  const proximities = data[0]?.proximities || [];
  const latitude = data[0]?.address.latitude || 0;
  const longitude = data[0]?.address.longitude || 0;
```